test_scripts/Defects/6_2/1384/common.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local runner = require('user_modules/script_runner')
+local hmi_values = require("user_modules/hmi_values")
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local m = {}
+
+--[[ Shared Functions ]]
+m.Title = runner.Title
+m.Step = runner.Step
+m.startOrigin = actions.start
+m.postconditions = actions.postconditions
+m.getPreloadedPT = actions.sdl.getPreloadedPT
+m.setPreloadedPT = actions.sdl.setPreloadedPT
+m.registerAppWOPTU = actions.app.registerNoPTU
+m.activateApp = actions.app.activate
+m.getMobileSession = actions.mobile.getSession
+m.getHMIConnection = actions.hmi.getConnection
+m.getConfigAppParams = actions.getConfigAppParams
+m.json = utils.json
+m.cloneTable = utils.cloneTable
+m.getAppEventName = commonRC.getAppEventName
+m.getAppRequestParams = commonRC.getAppRequestParams
+
+--[[ Common Variables ]]
+m.hmiExpectResponse = {
+  errorCodeWithAvailable = true,
```

Comment:
There is no reason for values in parameters, the array can contain only case names like
m.hmiExpectResponse = {
  "errorCodeWithAvailable",
  "errorWithoutAvailable"
}